Android/app/src/main/assets/currencies.json

```diff
@@ -0,0 +1 @@
id":"SHP"},"SCR":{"currencyName":"Seychellois Rupee","currencySymbol":"₨","id":"SCR"},"SBD":{"currencyName":"Solomon Islands Dollar","currencySymbol":"$","id":"SBD"},"LKR":{"currencyName":"Sri Lankan Rupee","currencySymbol":"₨","id":"LKR"},"THB":{"currencyName":"Thai Baht","currencySymbol":"฿","id":"THB"},"TRY":{"currencyName":"Turkish New Lira","id":"TRY"},"AED":{"currencyName":"UAE Dirham","id":"AED"},"VUV":{"currencyName":"Vanuatu Vatu","id":"VUV"},"YER":{"currencyName":"Yemeni Rial","currencySymbol":"﷼","id":"YER"},"AFN":{"currencyName":"Afghan Afghani","currencySymbol":"؋","id":"AFN"},"BDT":{"currencyName":"Bangladeshi Taka","id":"BDT"},"BRL":{"currencyName":"Brazilian Real","currencySymbol":"R$","id":"BRL"},"KHR":{"currencyName":"Cambodian Riel","currencySymbol":"៛","id":"KHR"},"KMF":{"currencyName":"Comorian Franc","id":"KMF"},"HRK":{"currencyName":"Croatian Kuna","currencySymbol":"kn","id":"HRK"},"DJF":{"currencyName":"Djiboutian Franc","id":"DJF"},"EGP":{"currencyName":"Egyptian Pound","currencySymbol":"£","id":"EGP"},"ETB":{"currencyName":"Ethiopian Birr","id":"ETB"},"XPF":{"currencyName":"CFP Franc","id":"XPF"},"GHS":{"currencyName":"Ghanaian Cedi","id":"GHS"},"GNF":{"currencyName":"Guinean Franc","id":"GNF"},"HKD":{"currencyName":"Hong Kong Dollar","currencySymbol":"$","id":"HKD"},"XDR":{"currencyName":"Special Drawing Rights","id":"XDR"},"KES":{"currencyName":"Kenyan Shilling","currencySymbol":"KSh","id":"KES"},"KGS":{"currencyName":"Kyrgyzstani Som","currencySymbol":"лв","id":"KGS"},"LRD":{"currencyName":"Liberian Dollar","currencySymbol":"$","id":"LRD"},"MOP":{"currencyName":"Macanese Pataca","id":"MOP"},"MVR":{"currencyName":"Maldivian Rufiyaa","id":"MVR"},"MXN":{"currencyName":"Mexican Peso","currencySymbol":"$","id":"MXN"},"NAD":{"currencyName":"Namibian Dollar","currencySymbol":"$","id":"NAD"},"NOK":{"currencyName":"Norwegian Krone","currencySymbol":"kr","id":"NOK"},"PLN":{"currencyName":"Polish Zloty","currencySymbol":"zł","id":"PLN"},"RUB":{"currencyName":"Russian Ruble","currencySymbol":"руб","id":"RUB"},"SZL":{"currencyName":"Swazi Lilangeni","id":"SZL"},"TJS":{"currencyName":"Tajikistani Somoni","id":"TJS"},"TTD":{"currencyName":"Trinidad and Tobago Dollar","currencySymbol":"TT$","id":"TTD"},"UGX":{"currencyName":"Ugandan Shilling","currencySymbol":"USh","id":"UGX"},"UYU":{"currencyName":"Uruguayan Peso","currencySymbol":"$U","id":"UYU"},"VND":{"currencyName":"Vietnamese Dong","currencySymbol":"₫","id":"VND"},"TND":{"currencyName":"Tunisian Dinar","id":"TND"},"UAH":{"currencyName":"Ukrainian Hryvnia","currencySymbol":"₴","id":"UAH"},"UZS":{"currencyName":"Uzbekistani Som","currencySymbol":"лв","id":"UZS"},"TMT":{"currencyName":"Turkmenistan Manat","id":"TMT"},"GBP":{"currencyName":"British Pound","currencySymbol":"£","id":"GBP"},"ZMW":{"currencyName":"Zambian Kwacha","id":"ZMW"},"BTC":{"currencyName":"Bitcoin","currencySymbol":"BTC","id":"BTC"},"BYN":{"currencyName":"New Belarusian Ruble","currencySymbol":"p.","id":"BYN"}}
```

Comment:
Do we really need this json file? Check the issue description, I have already added a link to get currency code. You can get that by default in java.

Reply:
Yes, I checked that, it returns only list of currency codes. We need currency name as well.